tests/system/feature_store/test_feature_store.py

```diff
@@ -4160,6 +4167,99 @@ def test_ingest_with_steps_drop_features(self):
         ):
             fstore.ingest(measurements, source)
 
+    @pytest.mark.skipif(
+        not is_databricks_configured(),
+        reason="databricks storage parameters not configured",
+    )
+    @pytest.mark.parametrize(
+        "source_class, target_class, local_file_name, reader, reader_kwargs",
+        [
+            (
+                CSVSource,
+                CSVTarget,
+                "testdata_short.csv",
+                pd.read_csv,
+                {"parse_dates": ["date_of_birth"]},
+            ),
+            (
+                ParquetSource,
+                ParquetTarget,
+                "testdata_short.parquet",
+                pd.read_parquet,
+                {},
+            ),
+        ],
+    )
+    def test_ingest_with_dbfs(
+        self, source_class, target_class, local_file_name, reader, reader_kwargs
+    ):
+        local_source_path = os.path.relpath(str(self.assets_path / local_file_name))
+        drop_column = "number"
+        key = "name"
+        test_dir = "/test_feature_store"
+        expected = reader(local_source_path, **reader_kwargs).drop(drop_column, axis=1)
+
+        measurements = fstore.FeatureSet("measurements", entities=[Entity(key)])
+
+        workspace = WorkspaceClient()
+        base_filename, extension = os.path.splitext(local_file_name)
+        generated_uuid = uuid.uuid4()
+        setup_dbfs_dirs(
+            workspace=workspace,
+            specific_test_class_dir=test_dir,
+            subdirs=[f'/{extension.replace(".", "")}_{generated_uuid}'],
+        )
+        dbfs_source_path = f"dbfs://{MLRUN_ROOT_DIR}{test_dir}/{generated_uuid}/source_{base_filename}{extension}"
+        dbfs_target_path = f"dbfs://{MLRUN_ROOT_DIR}{test_dir}/{generated_uuid}/target_{base_filename}{extension}"
+        with open(local_source_path, "rb") as source_file:
+            content = source_file.read()
+        with workspace.dbfs.open(dbfs_source_path, write=True, overwrite=True) as f:
+            f.write(content)
+
+        try:
```

Comment:
I guess this try needs to be moved up, so that it's immediately under the call to `setup_dbfs_dirs`. Or otherwise use test suite setup-teardown.